### PR TITLE
refactor: logs in record and test commands

### PR DESCRIPTION
**Pull request by ananupam**
## Related Issue
  -refactored logs from print to logger

Closes: #1202

#### Describe the changes you've made
Updated the codebase to utilize logging methods for generating logs instead of relying on print statements.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.

## Screenshots (if any)

|        Original         |          Updated           |
|:-----------------------:|:--------------------------:|
| 
image
| 
image
|

### CODE REVIEW

**Comment by ananupam** [comment]
I have read the CLA Document and I hereby sign the CLA

File: cmd/record.go [hunk 1]
```diff
@@ -189,8 +189,8 @@ func (r *Record) GetCmd() *cobra.Command {
 			}
 
 			if isDockerCmd && buildDelay <= 30*time.Second {
-				fmt.Printf("Warning: buildDelay is set to %v, incase your docker container takes more time to build use --buildDelay to set custom delay\n", buildDelay)
-				fmt.Println("Example usage:\n", `keploy record -c "docker-compose up --build" --buildDelay 35s\n`, "\nor\n", `keploy record -c "docker-compose up --build" --buildDelay 1m\n`)
+				r.logger.Warn(fmt.Sprintf("buildDelay is set to %v, incase your docker container takes more time to build use --buildDelay to set custom delay", buildDelay))
+				r.logger.Info(`Example usage: keploy record -c "docker-compose up --build" --buildDelay 35`)
```
Comment (gouravkrosx) [inline cmd/record.go:193]: `35s`

Comment (ananupam) [reply cmd/record.go]: Updated the same

**Review by gouravkrosx — APPROVED** [review]
LGTM

**Pull request merged**